packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/provision.template.bicep

```diff
@@ -0,0 +1,20 @@
+// Resources for bot
+module webAppProvision '\{{PluginIdPlaceholder.Provision.webapp.path}}' = {
+  name: 'webApp.Provision'
+  params: {
+    provisionParameters: provisionParameters
+    {{#if (contains "fx-resource-identity" plugins)}}
+    userAssignedIdentityId: \{{fx-resource-identity.References.identityResourceId}}
+    {{/if}}
+  }
+}
+
+output webAppOutput object = {
+  teamsFxPluginId: 'fx-resource-bot'
+  skuName: webAppProvision.outputs.botWebAppSKU
```

Comment:
Same here, do we need to add bot concept in this file. @hund030 @Siglud Let's discuss it offline